### ISSUE TITLE
[Task]: Playground frontend embedded integration tests

### ISSUE THREAD

**Issue by alexeyinkin** [apache/beam#24894]
### What needs to happen?

- Need to write all tests from #24728 except those not having the correspondent UI elements.

### Issue Priority

Priority: 3 (nice-to-have improvement)

### Issue Components

- [ ] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam examples
- [X] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

**Comment by alexeyinkin** [comment]
- Currently all testing of URLs other than '/' is blocked by the Flutter issue: https://github.com/flutter/flutter/issues/118028

**Comment by alexeyinkin** [comment]
.take-issue
